cmd/aws-s3-csi-controller/main.go

```diff
@@ -48,6 +49,7 @@ func main() {
 			Image:           *mountpointImage,
 			ImagePullPolicy: corev1.PullPolicy(*mountpointImagePullPolicy),
 		},
+		CSIDriverVersion: *csiDriverVersion,
```

Comment:
We can just get driver's version via: `version.GetVersion().DriverVersion`, using version package `github.com/awslabs/aws-s3-csi-driver/pkg/driver/version`

Reply:
I removed the new build-arg and used this instead, thanks!